http-gateway/service/config.go

```diff
@@ -130,9 +130,6 @@ func (c *OAuthClientConfig) Validate() error {
 	if c.ClientID == "" {
 		return fmt.Errorf("clientID('%v')", c.ClientID)
 	}
-	if c.Audience == "" {
```

Comment:
Commit message says about optional scopes but you remove audience. Is it correct?